administrator-manual/en/installation.rst

```diff
@@ -288,6 +288,15 @@ the name of the module as a parameter to the install script.  Example: ::
 
   nethserver-install nethserver-mail nethserver-nextcloud
 
+.. _installation-enterprise:
+
+Enterprise promotion
+====================
+
+To promote to |product| Enterprise version follow the istructions `available here`_.
```

Comment:
I'd change this to something like
"To promote |product| to the Enterprise version ..."